portfolio/src/main/webapp/script.js

```diff
@@ -36,7 +36,7 @@ function addFavoriteThing() {
 }
 // Retreive Messages from Servlet
 function getMessage() {
-  fetch('/data').then(response => response.json()).then(comments => {
+  fetch('/data?max-num-comments=2').then(response => response.json()).then(comments => {
```

Comment:
Not for this PR. But for capstone if we are doing something similar, read about pagination.